src/games/tictactoe/board.tsx

```diff
@@ -91,7 +95,7 @@ export class Board extends React.Component<IBoardProps, {}> {
   render() {
     if (this.props.ctx.gameover) {
       return (
-        <GameLayout gameOver={this._getGameOver()} />
+        <GameLayout gameOver={this._getGameOver()} playAgain={this.isAIGame() || this.isOfflineGame()} />
```

Comment:
Same, I don't like to have to repeat the same logic at each game, better to just pass the game type so there is no duplication of code